package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "flowcrypt-ios",
+  "version": "1.0.0",
+  "description": "Flowcrypt iOS app",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "prepare": "husky install"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FlowCrypt/flowcrypt-ios.git"
+  },
+  "author": "Flowcrypt",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/FlowCrypt/flowcrypt-ios/issues"
+  },
+  "homepage": "https://github.com/FlowCrypt/flowcrypt-ios#readme",
+  "devDependencies": {
+    "husky": "^8.0.1",
+    "lint-staged": "^12.4.3"
+  },
+  "lint-staged": {
+    "Core/*.{js,ts}": [
+      "cd Core && npm test",
+      "cd Core && npm run tslint",
+      "cd Core && npm run eslint"
+    ],
+    "appium/*.{js,ts}": [
+      "cd appium && npm run eslint"
```

Comment:
We can run `eslint --fix` before run `npm run eslint` for staged files to auto-fix eslint errors.
Do you want me to run `eslint --fix` before `npm run eslint`?

Reply:
on a precommit hook, sure

on semaphore ci, run linters without any attempts to fix, else they would mask problems

Reply:
ok